src/styles/blocks/cash.scss

```diff
@@ -0,0 +1,20 @@
+.cash {
+  display: flex;
+  height: 19px;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 16px;
+}
+
+.cash__name--price {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.cash__sum--value-of-price {
+  font-weight: 700;
+  font-size: 16px;
+  color: $standart-black-color;
```

Comment:
The variable '$standart-black-color' also has a typo. Consider renaming it to '$standard-black-color' for consistency and correct spelling.